src/components/common/AppInput.tsx

```diff
@@ -6,22 +6,26 @@ import {AppInputType} from '../../types/types';
 
 export const AppInput: React.FC<AppInputType> = ({
```

Comment:
Rename this component (from AppInput to something like CustomTextInput). The name is confusing.